src/video_utils.py

```diff
@@ -69,15 +70,10 @@ def get_video_duration(video_path):
         "format=duration",
         "-of",
         "default=noprint_wrappers=1:nokey=1",
-        video_path
+        video_path,
     ]
     try:
-        result = subprocess.run(
-            command,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            text=True, check=True
-        )
+        result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True, check=True)
```

Comment:
Line too long (111/100)Pylint[C0301:line-too-long](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/line-too-long.html)

Reply:
It's due to my preference of `max-line-length=120`, please find [here](https://github.com/yurnov/social-media-bot/blob/initial-docker/.github/workflows/linets.yml#L30).

What the `max-line-length` do you prefer?